_docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/paragraph/paragraph--get-started.html.twig

```diff
@@ -5,6 +5,8 @@ view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
 ] %}
 {% include  '@rhd/paragraph/download_thankyou.html.twig' %}
 
+{# Adding in katacoda #}
+<script src="//katacoda.com/embed.js"></script>
```

Comment:
This will need to be hard-coded to use the `https` endpoint from katacoda as the export process will transform this into a `http` version of the URL which will then result in insecure content warnings when the site is displayed to users over `https`.